Lib/cmd.py

```diff
@@ -297,13 +300,14 @@ def do_help(self, arg):
                 func = getattr(self, 'help_' + arg)
             except AttributeError:
                 try:
-                    doc=getattr(self, 'do_' + arg).__doc__
+                    doc = getattr(self, 'do_' + arg).__doc__
+                    doc = inspect.cleandoc(doc)
```

Comment:
IIUIC, this line is the actual patch

Reply:
Yeah, this and, obviously, the additional import